internal/protocols/fm1200/types.go

```diff
@@ -140,8 +140,7 @@ func (r *Record) ToProtobufDeviceStatus() *types.DeviceStatus {
 	info.Odometer = int32(r.Record.IOElement.Properties4B[TIO_OdmTotalMileage])
 	info.Position.Course = float32(r.Record.GPSElement.Angle)
 	info.Position.Satellites = int32(r.Record.IOElement.Properties1B[TIO_GSMSignal])
-	info.Temperature = float32(r.Record.IOElement.Properties4B[TIO_DallasTemperature])
-	info.AmbientTemperature = float32(r.Record.IOElement.Properties1B[TIO_AmbientAirTemperature])
+	info.Temperature = float32(r.Record.IOElement.Properties4B[TIO_DallasTemperature]) ||  float32(r.Record.IOElement.Properties1B[TIO_AmbientAirTemperature])
```

Comment:
since on front-end we process dallas temperature into celsius , then the ambient temperature should also be sent in dallas unit, or we will do the conversion on temperature which is already celsius